### DISCUSSION TITLE
Create a single bundle per page

### DISCUSSION THREAD

**Discussion by hspotted** [vercel/next.js#19132] [Help] [answered]
In the project i'm working one, we have limitations with the number of files we can handle.

So, imagine a scenario that we have 3 pages: Home, About, Contact Us. And the idea is when we build the project to export we only got, something like:
Home.html + Home.js
About.html + About.js
ContactUs.html + ContactUs.js

Is possible get that?

**Comment by jamesmosier** [comment, marked as answer]
That is not currently possible with Next. Next optimizes each page and there is no way to set the number of files you want to output. You may be able to tweak the webpack config a bit but that would likely break other parts of Next.

**Reply by hspotted** [reply]
thanks for the answer ;)
